src/probes/PrometheusClient.ts

```diff
@@ -47,11 +47,10 @@ export class PrometheusClient {
             if (series.length == 0) {
                 return null;
             }
-
             return series[0].value.value;
         } catch (error: unknown) {
             log.error('unable to query prometheus', { error });
-            return null;
+            return false;
```

Comment:
why not simply throw from here, instead of changing the return type?

the purpose of exception is exactly to avoid this kind of "error handling"...

Reply:
done here : https://github.com/scality/breakbeat/pull/11/commits/704fcfde37f2aac35b2ab0fec24b400d55b7c904#diff-ecb9e7c564c5a8b473a2faa0fbac5b456faf526b8a8034f2cdafc8e525c57a5eR40-R53